src/testsets.jl

```diff
@@ -15,22 +33,41 @@ flatted upon `finish`. See `bin/reporttests.jl` for an example of this use.
 and this is not recommended or supported.
 
 A `ReportingTestSet` has the `description` and `results` fields as per a
-`DefaultTestSet`, and has an additional `properties` field which is used
-to record properties to be inserted into the report.
+`DefaultTestSet`, and has the following additional fields:
+
+- `properties`: a dictionary which is used to record properties to be
+    inserted into the report.
+- `start_time::DateTime`: the start date and time of the testing (UTC).
+- `time_taken`: the time taken in seconds to run the `TestSet`.
+- `last_record_time`: the time when `record` was last called.
 
 See also: [`flatten_results!`](@ref), [`recordproperty`](@ref), [`report`](@ref)
 """
 mutable struct ReportingTestSet <: AbstractTestSet
     description::String
     results::Vector
     properties::Dict{String, Any}
+    start_time::DateTime
+    time_taken::Float64
+    last_record_time::Float64
```

Comment:
Why is this not:
```suggestion
    last_record_time::DateTime
```

Reply:
It wasn't to let the subtraction be straightforward, but can be now as it uses `Millisecond`